packages/@expo/package-manager/src/node/BasePackageManager.ts

```diff
@@ -67,17 +67,19 @@ export abstract class BasePackageManager implements PackageManager {
     return cwd;
   }
 
-  runAsync(command: string[]) {
+  runAsync(command: string[], options?: SpawnOptions) {
```

Comment:
We might need to default to `options: SpawnOptions = {}`, since it's being spread regardless if it's `undefined` or not 😄 

```suggestion
  runAsync(command: string[], options: SpawnOptions = {}) {
```

Reply:
the fun fact is that `{ ...undefined }` returns `{}`. but i think your even better and clear. will update accordingly. thanks!